docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -1596,18 +1606,19 @@ To facilitate the collection of features corresponding to the same compound ion
 
 - After the **ZipLoopEnd** node, add a **MapAlignerPoseClustering** node (**Community Nodes**>**OpenMS**>**Map Alignment**), set its Output Type to featureXML, and adjust the following settings:
 
-|**parameter**|	**value**|
-|:------------|:---------|
-|*algorithm* → *max_num_peaks_considered*|	−1|
-|*algorithm* → *superimposer* → *mz_pair_max_distance*|	0.005|
-|*algorithm* → *superimposer* → *num_used_points*|	10000|
-|*algorithm* → *pairfinder* → *distance_RT* → *max_difference*|	20.0|
-|*algorithm* → *pairfinder* → *distance_MZ* → *max_difference*|	20.0|
-|*algorithm* → *pairfinder* → *distance_MZ* → *unit*|	ppm|
+| **parameter**                                                 | **value** |
+| :------------------------------------------------------------ | :-------- |
+| _algorithm_ → _max_num_peaks_considered_                      | −1        |
```

Comment:
why have you changed these to italics? i think for everything menu options, bold is used.

Reply:
o right, these are not menu items but settings and values.